docs/articles/expensify-classic/workspace-and-domain-settings/Categories.md

```diff
@@ -1,5 +1,137 @@
+- Max Amount: Allows you to set specific expense amount caps based on the expense category. Using Limit type, you can define this per individual expense, or per day (for expenses in a category on an expense report).
+- Receipts: Allows you to decide whether you want to require receipts based on the category of the expense. For instance, it's common for companies to disable the receipt requirement for Toll expenses
+- Description: Allows you to decide whether to require the description field to be filled out based on the category of the expense
+- Description Hint: This allows you to place a hint in the description field. This will appear in light gray font on the expense edit screen in this field to prompt the expense creator to fill in the field accordingly.
+- Rule Enforcement: If users are in violation of these rules, those violations will be shown in red on the report. Any category-specific violations will only be shown once a category has been selected for a given expense. 
+
+
+## Make categories required
+This means all expenses must be coded with a Category. If they do not have a category, there will be a violation on the expense which can prevent submission. 
+## Update Category rules via spreadsheet
+You can update Category rules en masse by exporting them to CSV, editing the spreadsheet, and then importing them back to the categories page.
+
+This allows you to quickly add new categories and set GL codes, payroll codes, and description hints. This feature is only available for indirect integration setups. 
+
+## Category approvers
+Policy Admins can add additional approvers who must approve any expenses categorized with a particular category.
+
+To configure category approvers:
+
+1. Go to Settings > Workspace > Group > [Select Workspace] > Categories
+2. Click Edit Rules next to the category name that requires a category approver and use the "Approver" field
+   
+The expense will route to this Category approver, before following the approval workflow set up in the People table
+
+## Auto-categorize card expenses with default categories
+
+If you're importing card transactions, Default Categorization will provide a massive benefit to your company's workflow by automatically coding expenses to the proper GL.
+- Once configured according to your GL, the default category will detect the type of merchant for an expense based on its Merchant Category Code (MCC) and associate that expense with the proper GL account. 
+- This time-saver keeps employees from having to manually code expenses and provides admins with the peace of mind that the expenses coming in for approval are more reliably associated with the correct GL account. 
+- Best of all, this works for personal and company cards alike!
+  
+Setting up Default Categories
+
+1. First, go to Settings and select the group Workspace  that you want to configure
```

Comment:
Missing formatting:
**Settings**, **group Workspace**

Remove the extra blank after Workspace too so it shows:
**group Workspace** that you want
instead of 
group Workspace  that